app/code/community/Bolt/Boltpay/Helper/ApiTrait.php

```diff
@@ -214,18 +215,22 @@ protected function setCurlResultWithHeader($curlResource, $result)
         $this->curlHeaders = substr($result, 0, $curlHeaderSize);
         $this->curlBody = substr($result, $curlHeaderSize);
 
-        $this->setBoltTraceId();
+        $this->applyBoltTraceId();
     }
 
-    protected function setBoltTraceId()
+    /**
```

Comment:
Can you create an investigation task on the M2 board so we can look into this? We might forget otherwise.

Reply:
https://app.asana.com/0/941895179897714/1113616110414457